[2024-06-25T12:36:00.757+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T12:36:00.769+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: windturbine.group_check_temp.check_temp_branch manual__2024-06-25T12:35:46.893300+00:00 [queued]>
[2024-06-25T12:36:00.774+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: windturbine.group_check_temp.check_temp_branch manual__2024-06-25T12:35:46.893300+00:00 [queued]>
[2024-06-25T12:36:00.774+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-25T12:36:00.780+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): group_check_temp.check_temp_branch> on 2024-06-25 12:35:46.893300+00:00
[2024-06-25T12:36:00.786+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=916) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-25T12:36:00.786+0000] {standard_task_runner.py:63} INFO - Started process 918 to run task
[2024-06-25T12:36:00.787+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'windturbine', 'group_check_temp.check_temp_branch', 'manual__2024-06-25T12:35:46.893300+00:00', '--job-id', '465', '--raw', '--subdir', 'DAGS_FOLDER/windturbine.py', '--cfg-path', '/tmp/tmpl82r6ztg']
[2024-06-25T12:36:00.788+0000] {standard_task_runner.py:91} INFO - Job 465: Subtask group_check_temp.check_temp_branch
[2024-06-25T12:36:00.812+0000] {task_command.py:426} INFO - Running <TaskInstance: windturbine.group_check_temp.check_temp_branch manual__2024-06-25T12:35:46.893300+00:00 [running]> on host 8f35bebe2893
[2024-06-25T12:36:00.854+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='liviataina.ltab@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='windturbine' AIRFLOW_CTX_TASK_ID='group_check_temp.check_temp_branch' AIRFLOW_CTX_EXECUTION_DATE='2024-06-25T12:35:46.893300+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-25T12:35:46.893300+00:00'
[2024-06-25T12:36:00.855+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T12:36:00.862+0000] {baseoperator.py:400} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-25T12:36:00.867+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T12:36:00.868+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 269, in execute
    return self.do_branch(context, super().execute(context))
                                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/windturbine.py", line 53, in check_temperature
    if temperature >= 24:
       ^^^^^^^^^^^^^^^^^
TypeError: '>=' not supported between instances of 'str' and 'int'
[2024-06-25T12:36:00.872+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=windturbine, task_id=group_check_temp.check_temp_branch, run_id=manual__2024-06-25T12:35:46.893300+00:00, execution_date=20240625T123546, start_date=20240625T123600, end_date=20240625T123600
[2024-06-25T12:36:00.879+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/email.py:154: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2024-06-25T12:36:00.879+0000] {email.py:271} INFO - Email alerting: attempt 1
[2024-06-25T12:36:05.421+0000] {email.py:282} INFO - Sent an alert email to ['liviataina.ltab@gmail.com']
[2024-06-25T12:36:07.608+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 465 for task group_check_temp.check_temp_branch ('>=' not supported between instances of 'str' and 'int'; 918)
[2024-06-25T12:36:07.646+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-25T12:36:07.664+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-06-25T12:36:07.686+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-25T12:36:07.688+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
